src/app/zap-templates/zcl/data-model/chip/time-synchronization-cluster.xml

```diff
@@ -68,42 +101,81 @@ limitations under the License.
-    <attribute side="server" code="0x0008" define="TIME_ZONE_DATABASE" type="boolean" default="0" optional="true">TimeZoneDatabase</attribute>
-    <attribute side="server" code="0x0009" define="NTP_SERVER_PORT" type="INT16U" isNullable="true"  optional="true">NtpServerPort</attribute>
+    <attribute side="server" code="0x0008" define="TIME_ZONE_DATABASE" type="TimeZoneDatabaseEnum" default="3" optional="true">TimeZoneDatabase</attribute>
+    <attribute side="server" code="0x0009" define="NTP_SERVER_AVAILABLE" type="boolean" optional="true">NTPServerAvailable</attribute>
+    <attribute side="server" code="0x000A" define="TIME_ZONE_LIST_MAX_SIZE" type="INT8U" min="1" max="2" optional="true">TimeZoneListMaxSize</attribute>
+    <attribute side="server" code="0x000B" define="DST_OFFSET_LIST_MAX_SIZE" type="INT8U" min="1" optional="true">DSTOffsetListMaxSize</attribute>
+    <attribute side="server" code="0x000C" define="SUPPORTS_DNS_RESOLVE" type="boolean" optional="true">SupportsDNSResolve</attribute>
 
-    <!-- Test Commands -->
     <command source="client" code="0x00" name="SetUtcTime" optional="false">
-	<description>Upon receipt of this command, the server MAY update its UTCTime Attribute to match the time specified in the command</description>
-	<arg name="UtcTime" type="epoch_us"/>	
-	<arg name="Granularity" type="GranularityEnum"/>	
-	<arg name="TimeSource" type="TimeSourceEnum" optional="true"/>
-    </command>	
+      <description>This command MAY be issued by Administrator to set the time.</description>
+      <arg name="UtcTime" type="epoch_us"/>
+      <arg name="Granularity" type="GranularityEnum"/>
+      <arg name="TimeSource" type="TimeSourceEnum" optional="true"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x01" name="SetTrustedTimeSource" isFabricScoped="true" optional="false">
+      <description>This command SHALL set TrustedTimeSource.</description>
+      <arg name="TrustedTimeSource" type="FabricScopedTrustedTimeSourceStruct" isNullable="true"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x02" name="SetTimeZone" response="SetTimeZoneResponse" optional="true">
+      <description>This command SHALL set TimeZone.</description>
+      <arg name="TimeZone" type="TimeZoneStruct" array="true"/>
+      <access op="invoke" privilege="manage"/>
+    </command>
+
+    <command source="server" code="0x03" name="SetTimeZoneResponse" optional="true">
+      <description>Response to SetTimeZone.</description>
+      <arg name="DSTOffsetRequired" type="boolean"/>
+    </command>
+
+    <command source="client" code="0x04" name="SetDSTOffset" optional="true">
+      <description>This command SHALL set DSTOffset.</description>
+      <arg name="DSTOffset" type="DSTOffsetStruct" array="true"/>
```

Comment:
Again, this field id is 1 in the spec but 0 here.

Reply:
Same, there is open PR.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/6484